src/muiComponents/layout/Container/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { AppBar, Toolbar } from "@mui/material"
+import React, { PropsWithChildren, ReactElement } from "react"
+import MuiContainer from "@mui/material/Container/Container"
+import { Switch } from "@mui/material"
+import { ThemeMode } from "../../../providers/SettingContext"
+import useSettings from "../../../hooks/useSetting"
+
+export default function Container({
+  children,
+}: PropsWithChildren<unknown>): ReactElement {
+  const { themeMode, onChangeMode } = useSettings()
+  const themeSwitchChecked = themeMode === "dark" ?? "light"
```

Comment:
This name is hard for me to understand. Could `currentThemeSelected` describe this?